xbmc/settings/Settings.h

```diff
@@ -335,6 +335,7 @@ class CSettings : public CSettingsBase, public CSettingCreator, public CSettingC
   static const std::string SETTING_INPUT_PERIPHERALS;
   static const std::string SETTING_INPUT_PERIPHERALLIBRARIES;
   static const std::string SETTING_INPUT_ENABLEMOUSE;
+  static const std::string SETTING_INPUT_IRSERVERSUITE;
```

Comment:
As this is only needed at one place you may consider moving it out of this file.
see https://github.com/xbmc/xbmc/pull/13843#discussion_r186252896

Reply:
Pls. provide a solution

Reply:
you can see what I'm suggesting at #13856

Reply:
@ksooo ^^

Reply:
I'm against removing the constant. Today the new setting is only used at one place, yes. But if somebody starts using it at another place, there are good chances that we forget to introduce the constant. We have a simple and working rule: add a constant for a new setting. There is no drawback with this, imo. Just my 2 ct, though.

@peak3d I do really very sorry for all the trouble you get with this simple PR.